media/js/firefox/android.js

```diff
@@ -283,85 +288,21 @@
         'data-download-version': 'href'
     });
 
-    // send-to-device test variations
-    var sendToDeviceWidget;
-    var $sendToDeviceWidgetForm;
-    var $sendToDeviceModal;
-    var $sendToDeviceEmbeddedWidget;
-    var $sendToLinks = $('.send-to');
-
-    // data-* stored in #intro element to determine test configuration
-    // contains two properties: lang (page language) and variant ('a', 'b', 'c', '')
-    var testVars = $('#intro').data();
-
-    var initSendToDeviceForm = function() {
+    if (hasWidget) {
+        sendToDeviceWidgetTop = $('#send-to-device').offset().top;
         sendToDeviceWidget = new Mozilla.SendToDevice();
         sendToDeviceWidget.init();
-    };
-
-    // set up default behavior for send to device widget - opens in modal
-    // used for no variant (no test) and variant "a"
-    var defaultSendToDevice = function() {
-        // add class to display the widget
-        $('#send-to-device-default').removeClass('send-to-device-variant');
 
-        // reference modal wrapper for widget
-        $sendToDeviceModal = $('#send-to-modal-container');
+        // add hidden input to update basket id
+        $('#send-to-device-form').append('<input type="hidden" name="send-to-device-basket-id" value="android-embed">');
```

Comment:
If `android-embed` is now the default basket ID for this form, would it be better in the macro itself as opposed to added here in JS? I'm assuming this is triggering a specific email too, in which case two questions come to mind:

1.) Is it translated into all the languages we need to make this default?
2.) If so, should we just make this email the default basket ID for android, instead of sticking with the name `android-embed`? Would this be too much of a pain to make renaming worthwhile?